scripts/workers/code_upload_submission_worker.py

```diff
@@ -582,6 +591,8 @@ def main():
         "submission_time_limit"
     )
     while True:
+        if challenge.get("is_static_dataset_code_upload"):
+            time.sleep(2.1)
```

Comment:
@savish28 Please add a comment here, why this is necessary.

Reply:
@KhalidRmb sure done.